src/Microsoft.Data.Sqlite/Utilities/SqliteVersion.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Data.Sqlite.Interop;
+
+namespace Microsoft.Data.Sqlite.Utilities
+{
+    internal class SqliteVersion
+    {
+        public readonly static Version Current = new Version(NativeMethods.sqlite3_libversion());
+    }
+}
```

Comment:
'Tis a silly class. I like the idea of putting a public static `SQLITE_VERSION` field on `NativeMethods` that gets initialized in the static constructor. `SqliteConnection.ServerVersion` should be updated to use it. Then again, does the value even need to be cached?